servers/audio_server.h

```diff
@@ -56,15 +56,26 @@ class AudioDriver {
 	SafeNumeric<uint64_t> prof_time;
 #endif
 
+public:
+	struct SizePosition {
+		unsigned int size;
+		unsigned int position;
+
+		SizePosition(unsigned int p_size = 0, unsigned int p_position = 0) noexcept :
```

Comment:
`noexcept` to fix compilation error with `g++`
Copied from: https://github.com/godotengine/godot/pull/92969#discussion_r1634465242.